build.yaml

```diff
@@ -1602,6 +1621,7 @@ steps:
      - base_image
      - build_site_www
      - hail_ubuntu_image
+     - deploy_hail
```

Comment:
gotta be deploy, otherwise the latest-hash will not be up-to-date if this is a build of a commit that changes the PyPI version.